client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -20,6 +30,57 @@ import EventSponsorsForm from './EventSponsorsForm';
 import EventVenueForm from './EventVenueForm';
 import { EventFormProps, fields, EventFormData } from './EventFormUtils';
 
+interface IntegrationInfoData {
+  isAuthenticated: boolean | null | undefined;
+  hasCalendar: boolean;
+}
+
+const IntegrationInfo = ({
+  isAuthenticated,
+  hasCalendar,
+}: IntegrationInfoData) => {
+  const infoData = ({ isAuthenticated, hasCalendar }: IntegrationInfoData) => {
+    if (isAuthenticated) {
+      if (hasCalendar) {
+        return {
+          Icon: InfoIcon,
+          text: 'Instance is authenticated with calendar api, and chapter has created calendar. Chapter will attempt creating event in the calendar.',
+        };
+      }
+      return {
+        Icon: WarningTwoIcon,
+        text: "Instance is authenticated with calendar api, but chapter doesn't have calendar created. Event will not be created in the calendar.",
```

Comment:
How about the following?

"This chapter does not have a calendar. To allow calendar event creation, go to `link-to-chapter` and create a calendar"